### DISCUSSION TITLE
V100 16GB why not 32?

### DISCUSSION THREAD

**Discussion by Amit30swgoh** [lllyasviel/Fooocus#2364] [Q&A] [answered]
![image](https://github.com/lllyasviel/Fooocus/assets/42430868/9fac2745-50e1-40bf-9d61-baf0eeffff76)
![image](https://github.com/lllyasviel/Fooocus/assets/42430868/95e6affa-6d1b-48fc-af21-aa6944e1b77d)
Thanks for the info

**Comment by mashb1t** [comment]
What is the question? Colab only offers V100 with 16GB VRAM i assume, if that's what you mean

**Reply by Amit30swgoh** [reply, marked as answer]
So why is it better than t4? It cost 5.45H compared to T4 2.05 ( both 50GB ram instead of 12).
Altho even tho it's only 16GB compared to 15GB T4, it's faster, I would say around 50%-70% faster, but again, it costs twice compute units

** edit OK after reading this I think it's OK volta100 much better :) basically the more Cuda cores the faster it will generate... Thanks closing
https://www.topcpu.net/en/gpu-c/tesla-t4-vs-tesla-v100-dgxs-16-gb
